client/src/App.tsx

```diff
@@ -6,12 +6,48 @@ type Props = {
     drizzle: Drizzle,
 };
 
-const App: React.FC<Props> = ({ drizzle }) => {
-    return (
-        <div className="App">
-            Hell World
-        </div>
-    )
-};
+type State = {
+    loading: boolean,
+    drizzle: Drizzle,
+    unsubscribe: Function,
+}
+
+class App extends React.Component<Props> {
+    state: State;
```

Comment:
You should pass state as the second type arg after Props:

```tsx
React.Component<Props, State>
```

You could also use hooks instead of a stateful component